Spanner/src/Connection/Grpc.php

```diff
@@ -981,6 +990,14 @@ public function streamingRead(array $args)
             );
         }
 
+        $directedReadOptions = $this->pluck('directedReadOptions', $args, false);
+        if (!empty($directedReadOptions)) {
+            $args['directedReadOptions'] = $this->serializer->decodeMessage(
+                new DirectedReadOptions,
+                $directedReadOptions
+            );
+        }
```

Comment:
make a function similar to `instanceConfigObject` to avoid code duplication

Reply:
Done